lesson_6/Array.js

```diff
@@ -0,0 +1,29 @@
+let lenghtArray = Number(prompt('Type lenght of array: '));
+const newArrayFromUser = new Array();
+console.log(newArrayFromUser);
+
+
+if (lenghtArray === 0) {
+  console.log(`You have empty array with lenght ${lenghtArray}`);
+}
+
+
+while (lenghtArray > 0) {
+  const arrayValue = prompt('Type element of array: ');
+  if (parseInt(arrayValue)) {
+    newArrayFromUser.push(+arrayValue);
+  } else {
+    newArrayFromUser.push(arrayValue);
+  }
+  lenghtArray -= 1;
+  console.log(newArrayFromUser);
+}
+console.log(newArrayFromUser);
+
+
+newArrayFromUser.sort();
```

Comment:
default `sort()` method will sort data like a `string`. Check the result with array, for example `[1, 2, 3, 12, 24, 41]`

Reply:
done